option optimize_for = LITE_RUNTIME;

message EquipRefiningRequest
{
 required uint32 posZhengXing = 1;
 required uint32 pos = 2;
 repeated uint32 itemId = 3;
 repeated uint32 itemNum = 4;
}
